deps/blas.mk

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS :=
+OPENBLAS_FFLAGS := $(FFLAGS) $(JFFLAGS)
 OPENBLAS_CFLAGS :=
```

Comment:
And if there's a problem with FFLAGS, then it's likely that being inconsistent and not doing the same thing with CFLAGS is still an issue. This is setting CFLAGS to empty everywhere except 32 bit windows.